proposals/hasCrossSiteAncestor.md

```diff
@@ -104,17 +104,21 @@ string. A UUID of the document.
 
 | Required Combinations | Notes |
 |---|---|
-|frameId | frameId must be a non-zero value|
 |tabId + frameId |frameId can be 0|
 |tabId | will use the top-level frame (frameId 0)|
+|frameId| value must not be 0|
 |documentId| tabId/frameId are not required but permitted|
 
 ##### Return value
 A Promise that will be fulfilled with a `Cookie.partitionKey` object that matches the properties given in the details parameter and contains the `hasCrossSiteAncestor` value associated with the current cross-site status of the frame.
 
+##### Permissions required to use Cookies.getPartitionKey()
+- Host permissions for the frame's document.
+  
 ##### Error conditions
 - If host permissions are not granted for the document whose partitionkey is getting queried an error will be returned.
 - When the parameters passed do not correspond to an existing frame, an error will be returned.
+- If the partitionKey that would be associated with the frame can not be serialized an error will be returned. This can happen when the origin associated with the topLevelSite is opaque or if the underlying key associated with the frame has a nonce.
```

Comment:
Suggested edit in relation to the discussion from https://github.com/w3c/webextensions/pull/712#discussion_r1822980547

```suggestion
- If the partitionKey that would be associated with the frame can not be serialized an error will be returned. This can happen when the origin associated with the topLevelSite is opaque or if the underlying key associated with the frame has a nonce. This is comparable to situations where `document.cookie` would throw.
```